.github/workflows/make-plugin-arm.yml

```diff
@@ -0,0 +1,39 @@
+name: Make ARM Plugins
+
+on:
+  workflow_dispatch:
+    inputs:
+      version_number:
+        description: 'Version number'
+        required: true
+        default: '1.x.x'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: 'recursive'
+    - uses: goto-bus-stop/setup-zig@v1
+      with:
+        version: 0.9.1
+    - name: Build plugin
+      env:
+        VERSION: ${{ github.event.inputs.version_number }}
+      run: |
+        cd javascript/net/grpc/web/generator
+        zig build -Drelease-fast
+    - name: gen and verify sha256
+      run: |
+        cd javascript/net/grpc/web/generator/zig-out/bin
+        for exe in $(ls)
+        do
+          openssl dgst -sha256 -r -out $exe'.sha256' $exe
+          sha256sum -c $exe'.sha256'
+        done
```

Comment:
Do you mind doing an additional verification step like the linux workflow? thanks!

https://github.com/grpc/grpc-web/blob/master/.github/workflows/make-plugin-linux.yml

Reply:
I could do that, but is there any reason for that?

Reply:
just to keep consistency with the existing flows.. (it's a separate discussion whether it's useful at all.. if it's not we should remove it across the board, but the easier thing to do is probably to just add it :))

Reply:
The existing flows only build one binary once, however, we are now building 3 binaries at once in the new flow, and the files may look like this after build:

```
├── BUILD.bazel
├── build.zig
├── grpc_generator.cc
├── Makefile
└── zig-out
   └── bin
      ├── protoc-gen-grpc-web-unknown-darwin-aarch64
      ├── protoc-gen-grpc-web-unknown-linux-aarch64
      └── protoc-gen-grpc-web-unknown-windows-aarch64.exe
```

Instead of specifying the 3 binary names explicitly, I use a BASH for loop here `cd javascript/net/grpc/web/generator/zig-out/bin && for exe in $(ls)` (considering we may support more platforms in the future). 

If we have to do verification in an additional step, we may have to add one more FOR LOOP like this `for exe in $(ls | grep -v sha256)`, which may contain a potential problem that we are assuming the binary file name doesn't contain `sha256`. It doesn't cause any issues for now but I'm worried about the future, especially when we want to support more platforms.

This is why I prefer to generate sha256 and verify it in one loop.

Do you have any suggestions for me?